Frontend/lib/features/app_router/app_router.dart

```diff
@@ -228,16 +255,9 @@ final router = GoRouter(
                   GoRoute(
                     path: '/matches',
                     builder: (context, state) {
-                      return const UpcomingMatchesPage();
+                      return const MatchListWidget();
```

Comment:
Kevin heeft recentelijk allerlei aanpassingen gedaan aan de UpcomingMatchesPage om ook tournaments juist weer te geven. Deze moeten we dus eigenlijk wel tonen, omdat je hier ook tournaments aanmaakt en matches kan editen, maar ik snap ook waarom je hier de MatchListWidget laat zien, want we willen ook pending/active matches laten zien (en spelen op desktop).

Ik denk dat hier nog een beslissing over gemaakt moet worden, want nu zijn er in principe twee pagina's die hier weergegeven moeten worden, beide zo belangrijk, maar met verschillende functionaliteit. Kunnen ze samengevoegd worden op een manier?